core/revised/combat.tex

```diff
@@ -189,4 +189,28 @@ \subsection{Healing}\label{subsec:heal}
 Abilities like Time Mage’s Wild Magic also just roll a d10. Dervish's Deadly Accuracy is essentially tripling your critical hit rate, so they can threaten a critical on an 8, 9, or 10, and then resolve the confirmation roll as other classes would. The Rogue’s Dice deals damage equal to 10 times the d10 results. The Berserker’s Unwavering Fury Ability can increase or decrease the confirmation roll’s value by 1.
 
 This option swaps granularity for speed, reducing the mental load on the GM and the players. It is recommended to use it if you don’t mind losing the granularity of the d100 and/or the nostalgic experience of using d100, especially if you do not have access or do not want to use computer aid during the game.
-\end{multiboco}
\ No newline at end of file
+\end{multiboco}
+
+\begin{multicols}{2}
+\subsection{Special Combat Rules}\label{misccombat}
+This section lists several miscellaneous rules that apply to combat encounters. They are corner cases that may not apply to every combat, but can clarify special situations.
+
+\textbf{Opposing Status Effects}: When a character is subject to multiple status effects with opposed effects, as for example having \tstatus{Curse} and \tstatus{Blink}, they cancel each other. Should their duration be different, the status that lasts longer reinstate its effects after the shorter status ends.
+
+\textbf{Multiple Elemental Status Effects}: When a character is subject to multiple instances of \tstatus{Vulnerable}, \tstatus{Resist}, \tstatus{Immune} and \tstatus{Absorb} status effects to the same element (say, a character with \tstatus{Immune:Ice} and \tstatus{Vulnerable:Ice}), follow the following order: Apply \tstatus{Immune}, then \tstatus{Absorb}. Should neither be present, apply \tstatus{Vulnerable} and/or \tstatus{Resist} (which are considered opposed, as per the \textbf{Opposing Status Effects} rule).  
+
+\textbf{Multiple Rerolls}: When multiple rerolls affect the same action, "positive" rerolls (roll twice and take best) and "negative" rerolls (roll twice and take worst) cancel themselves. Should there be any rerolls left after that, the character apply them as normal. 
```

Comment:
Quotes should use the csquotes quote macros.

Reply:
cf95de8